java.lang.NullPointerException:    at openfoodfacts.github.scrachx.openfood.views.MainActivity.detectBarCodeInImage

```
Caused by: java.lang.NullPointerException: 
  at openfoodfacts.github.scrachx.openfood.views.MainActivity.detectBarCodeInImage (MainActivity.java)
  or                     .lambda$myContributions$5$MainActivity (MainActivity.java)
  at openfoodfacts.github.scrachx.openfood.views.MainActivity.access$000 (MainActivity.java)
  or                     .changeFragment (MainActivity.java)
  or                     .chooseDialog (MainActivity.java)
  or                     .createAlertDialog (MainActivity.java)
  or                     .lambda$myContributions$6$MainActivity (MainActivity.java)
  or                     .lambda$onCreate$0$MainActivity (MainActivity.java)
  or                     .lambda$onCreate$3$MainActivity (MainActivity.java)
  or                     .lambda$scan$4$MainActivity (MainActivity.java)
  or                     .setItemSelected (MainActivity.java)
  at openfoodfacts.github.scrachx.openfood.views.MainActivity.handleSendImage (MainActivity.java)
  or                     .lambda$null$1$MainActivity (MainActivity.java)
  at openfoodfacts.github.scrachx.openfood.views.MainActivity.handleIntent (MainActivity.java)
  or                     .lambda$null$2$MainActivity (MainActivity.java)
  or                     .updateBadgeOfflineEditDrawerITem (MainActivity.java)
  at openfoodfacts.github.scrachx.openfood.views.MainActivity.onCreate (MainActivity.java)
  at android.app.Activity.performCreate (Activity.java:6910)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1123)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2746)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2864)
  at android.app.ActivityThread.-wrap12 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1567)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:156)
  at android.app.ActivityThread.main (ActivityThread.java:6523)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:942)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:832)
```
![image](https://user-images.githubusercontent.com/1689815/54600456-1f4b4d80-4a3d-11e9-9efd-8a13e6004cf7.png)

--- a/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java
+++ b/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java
@@ -73,6 +73,7 @@ import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.InputStream;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.Hashtable;
 import java.util.Objects;
 
@@ -899,7 +900,6 @@ public class MainActivity extends BaseActivity implements CustomTabActivityHelpe
         Uri selectedImage = null;
         ArrayList<Uri> selectedImagesArray = new ArrayList<>();
         selectedImage = intent.getParcelableExtra(Intent.EXTRA_STREAM);
-        boolean isBarCodePresent = false;
         if (selectedImage != null) {
             selectedImagesArray.add(selectedImage);
             chooseDialog(selectedImagesArray);
@@ -909,6 +909,7 @@ public class MainActivity extends BaseActivity implements CustomTabActivityHelpe
     private void handleSendMultipleImages(Intent intent) {
         ArrayList<Uri> selectedImagesArray = intent.getParcelableArrayListExtra(Intent.EXTRA_STREAM);
         if (selectedImagesArray != null) {
+            selectedImagesArray.removeAll(Collections.singleton(null));
             chooseDialog(selectedImagesArray);
         }
     }

